core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -3868,6 +3868,92 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
     } finally client.close(time.Duration.ZERO)
   }
 
+  @Test
+  def testListConfigResources(): Unit = {
+    client = createAdminClient
+
+    // Alter group and client metric config to add group and client metric config resource
+    val clientMetric = "client-metrics"
+    val group = "group"
+    val clientMetricResource = new ConfigResource(ConfigResource.Type.CLIENT_METRICS, clientMetric)
+    val groupResource = new ConfigResource(ConfigResource.Type.GROUP, group)
+    val alterResult = client.incrementalAlterConfigs(util.Map.of(
```

Comment:
nit: Can't this just be `Map.of`?

Reply:
This is a scala file and I would like to avoid java/scala transfer, so I use java map here. Same as java set above.

Reply:
Ah, yes. Sorry.